main.go

```diff
@@ -75,26 +79,46 @@ var (
 		},
 		[]string{"remote"},
 	)
+	httpRequestDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "http_request_duration_ms",
+			Help:    "Duration of HTTP request in milliseconds",
+			Buckets: prometheus.DefBuckets,
+		},
+		[]string{"path"},
+	)
+	writeThroughtput = util.NewThroughputCalc(tickInterval)
 )
 
 func init() {
 	prometheus.MustRegister(receivedSamples)
 	prometheus.MustRegister(sentSamples)
 	prometheus.MustRegister(failedSamples)
 	prometheus.MustRegister(sentBatchDuration)
+	prometheus.MustRegister(httpRequestDuration)
+	writeThroughtput.Start()
 }
 
 func main() {
 	cfg := parseFlags()
 	http.Handle(cfg.telemetryPath, prometheus.Handler())
-	logLevel := promlog.AllowedLevel{}
-	logLevel.Set(cfg.logLevel)
-	logger := promlog.New(logLevel)
+	util.InitLogger(cfg.logLevel)
+
+	writer, reader := buildClients(cfg)
+
+	http.Handle("/write", timeHandler("write", write(writer)))
+	http.Handle("/read", timeHandler("read", read(reader)))
+	http.Handle("/healthz", health(reader))
 
-	writers, readers := buildClients(logger, cfg)
+	util.LogInfo("msg", "Starting up...")
```

Comment:
wondering if it's cleaner to put this in our own `log` or `logger` package so that you can do `log.Info()`, `log.Debug()`, etc.?